full-service/src/json_rpc/v1/api/wallet.rs

```diff
@@ -181,6 +181,7 @@ where
                         subaddress_index: None,
                     },
                     None,
+                    None, // Note: Not including spend_only_from_subaddress in V1 API
```

Comment:
Needed to update the called method from the v1 API, but we are not adding this functionality to V1.